### PR TITLE
Update Replay images to reflect new layout

### PR DESCRIPTION
**Pull request by jas-kas**
- Update Replay images to reflect new layout
- Update messaging on Replay Details to reflect new layout (Dropped "DOM Events" section and incorporated some of that text in the Breadcrumbs section)

### CODE REVIEW

File: src/docs/product/session-replay/replay-details.mdx [hunk 1]
```diff
@@ -13,7 +13,7 @@ Every replay has a detailed view that contains the embedded video player and ric
...
   - Some [breadcrumb types](/product/issues/issue-details/breadcrumbs/) visible in **Issue Details** are not 1:1 to the replay breadcrumbs list. The trail of events typically seen in the Issue Details page are now displayed in the Console and Network components of the **Replay Details** page.
 
 - **Timeline:** This is the section at the top of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events.
```
Comment (c298lee) [inline src/docs/product/session-replay/replay-details.mdx:19]:
```suggestion
- **Timeline:** This is the section at the bottom of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events.
```
Maybe we can mention that you can zoom into the timeline too?

Comment (jas-kas) [reply src/docs/product/session-replay/replay-details.mdx:19]: @c298lee Yes, please leave a commit suggestion for that!

File: src/docs/product/session-replay/replay-details.mdx [hunk 2]
```diff
@@ -13,17 +13,15 @@ Every replay has a detailed view that contains the embedded video player and ric
...
   - Some [breadcrumb types](/product/issues/issue-details/breadcrumbs/) visible in **Issue Details** are not 1:1 to the replay breadcrumbs list. The trail of events typically seen in the Issue Details page are now displayed in the Console and Network components of the **Replay Details** page.
 
-- **Timeline:** This is the section at the top of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events.
+- **Timeline:** This is the section at the bottom of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events.
```
Comment (c298lee) [inline src/docs/product/session-replay/replay-details.mdx:19]:
```suggestion
- **Timeline:** This is the section at the bottom of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events and has zoom functionality to easily distinguish between events that are near in time.
```

**Review by c298lee — APPROVED** [review]

File: src/docs/product/session-replay/replay-details.mdx [hunk 3]
```diff
@@ -13,17 +13,15 @@ Every replay has a detailed view that contains the embedded video player and ric
...
   - Some [breadcrumb types](/product/issues/issue-details/breadcrumbs/) visible in **Issue Details** are not 1:1 to the replay breadcrumbs list. The trail of events typically seen in the Issue Details page are now displayed in the Console and Network components of the **Replay Details** page.
 
-- **Timeline:** This is the section at the top of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events.
+- **Timeline:** This is the section at the bottom of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events and has zoom functionality to easily distinguish between events that are near in time.
```
Comment (jas-kas) [inline src/docs/product/session-replay/replay-details.mdx:19]:
```suggestion
- **Timeline:** This is the section at the bottom of the **Replay Details** page that illustrates where significant events (such as errors, rage clicks, and user interactions) happen over the course of the replay. This allows users to easily scrub to key events by dragging across the timeline. It also visually conveys the amount of time that took place between events and has a zoom functionality so you can easily zoom-in to distinguish between events that happen close together.
```

**Pull request merged**